Kickstarter-iOS/Views/Controllers/PledgeViewControllerTests.swift

```diff
@@ -100,11 +108,15 @@ final class PledgeViewControllerTests: TestCase {
   }
 
   func testView_PledgeContext_NeedsConversion_IsFalse() {
-    let userEnvelope = UserEnvelope(me: GraphUserCreditCard.withCards([
-      GraphUserCreditCard.visa,
-      GraphUserCreditCard.masterCard
-    ]))
-    let mockService = MockService(fetchGraphCreditCardsResponse: userEnvelope)
+    let cardTemplate = GraphUserCreditCard(
+      nodes: [
+        GraphUserCreditCard.visa,
+        GraphUserCreditCard.masterCard
+      ]
+    )
+    let withCards = GraphUser.template |> \.storedCards .~ cardTemplate
+    let response = UserEnvelope<GraphUser>(me: withCards)
```

Comment:
Actually we seem to be using the same `response` a lot, can't you make it a global constant to the file?